docs: Describing how to run acceptance scenario (#817)
scaleway/terraform-provider-scaleway: TESTING.md

# Testing the module

## Unit testing

> In order to test the provider, you can simply run `make test`.

Reviewer: ```suggestion
You can test the provider, by running `make test`.
```
